src/components/Aside/Aside.js

```diff
@@ -1,7 +1,36 @@
 import React from "react";
-
-const Aside = () => {
-  return <div>Aside</div>;
+import NavSideCategoryComponents from "../Nav/NavSideCategoryComponents";
+import "./Aside.scss";
+const Aside = ({ categoryList, powerOn, handleSideBarOn }) => {
```

Comment:
```suggestion
import "./Aside.scss";

const Aside = ({ categoryList, powerOn, handleSideBarOn }) => {
```

* 가독성을 위해 import 구문과 component 시작점을 한줄 띄어 주세요